jetson/odrive_bridge/src/__main__.py

```diff
@@ -329,7 +329,7 @@ def drive_vel_cmd_callback(channel, msg):
             global left_speed, right_speed
 
             speed_lock.acquire()
-            left_speed, right_speed = cmd.left, cmd.right
+            left_speed, right_speed = -cmd.left, -cmd.right,

```

Comment:
can you leave this as cmd.left and cmd.right and just flip the negative sign in modrive.set_vel?